packages/cli/src/cliEntry.js

```diff
@@ -138,11 +138,19 @@ async function run() {
 }
 
 async function setupAndRun() {
-  const setupEnvScript = /^win/.test(process.platform)
-    ? path.join('..', 'setup_env.bat')
-    : path.join('..', 'setup_env.sh');
-
-  childProcess.execFileSync(path.join(__dirname, setupEnvScript));
+  // We only have a setup script for UNIX envs currently
+  if (/^win/.test(process.platform) === false) {
```

Comment:
This seems like a shady test. Is there something that is not regex that would be more reliable?

Reply:
Loads better! With your change, if there is an OS variant of linux coming around that they call `linuxIsNotWin`, we'd be screwed.